backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs

```diff
@@ -19,7 +19,15 @@ public async Task<List<MeetingThreeMembersDTO>> GetThreeMeetingMembersAsync()
                     .ThenInclude(meetingMember => meetingMember.TeamMember)
                     .ThenInclude(teamMember => teamMember.User)
                 .ToListAsync();
-
+            
+            if (teamId is null)
+            {
+                return new List<MeetingThreeMembersDTO>();
+            }
+            
+            var team = await _context.Teams.FirstOrDefaultAsync(team => team.Id == teamId) ?? throw new KeyNotFoundException("Team doesn't exist");
+            
+            meetings = meetings.Where(meeting => meeting.TeamId == team.Id).ToList();
```

Comment:
It would be better to filter before loading

Reply:
Fixed